docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

```diff
@@ -8,14 +9,79 @@ redirect_from:
+
+{% endinfo_block %}
+
+## Three weeks before go-live
+
+- **Verify that your deploy.yml file is set up correctly**. Verify that your project works and works with the final endpoints. You can set both testing and final endpoints in your deploy.yml file. Your developers can now mock a "live" operation of the project with its final endpoints by adjusting their local host entries and test your shop thoroughly.
+- **TLS certificates to be used are provisioned**. In case of delegation DNS to Spryker, TLS certificates for your endpoints should be created automatically. If you want us to create a TLS certificate for your endpoints but do not want to delegate your DNS, we can provide you with the verification records. If you have not received them already, contact us using the [Support Portal](https://support.spryker.com). If you do not delegate your DNS, make sure to provide us with the TLS certificates as detailed in the [Setting up a custom SSL certificate](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/setting-up-a-custom-ssl-certificate.html#next-step) instruction so we can configure the certificates for you.
+Make sure to deploy to your production environment and run checks regularly—see [Deploying in a production environment](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/deploying-in-a-production-environment.html).
+
+## Two weeks before go-live
+
+- **Remove all demo data from your environment**. Ensure that you exclusively work with real data, which will be used after the go-live. Remove all demo data that comes with the Spryker repository, which explicitly includes all demo users (as well as the standard admin user), because having demo users active poses a significant security risk for your project.
+- **Communicate your go-live plan to Spryker**. Make sure to reach out to your Partner or Customer Success Manager and share your go-live plans—the day and time when you want to make your shop accessible to the public. If this time should change, keep us updated. This is mission-critical for DNS switching and the hyper care phase, which Spryker Support provides for you before and after your go-live.
+
+## One week before go-live
+
+- **Double-check the go-live date**. If any of the preceding tasks remain uncompleted, postpone your go-live, or discuss with us how to complete them in time.
+
+{% info_block infoBox "Note" %}
+		
+All DNS-related topics are affected by delays that are inherent to the DNS system, and we cannot apply DNS changes or validations any faster. 
+
+{% endinfo_block %}
+
+{% info_block infoBox "Note" %}
+	
+If you find yourself struggling with the preceding points, reach out to Spryker Support using your Onboarding case **immediately**.
+	
+{% endinfo_block %}
```

Comment:
Two concurrent notes are distracting, consider making one note as a plain text, or move any of these notes to another place, so there are no notes that go one after the other